html/changelogs/lohikar-lighting.yml

```diff
@@ -1,7 +1,7 @@
 author: Lohikar
 delete-after: True
 changes: 
```

Comment:
If you read the PR summary, you'll see that I forgot the directional lighting changelog in the first place and was re-adding it.
The other one was just a tweak to the existing lighting changelog.

100% intentional.